app/services/irs_attempts_api/tracker_events.rb

```diff
@@ -156,13 +156,13 @@ def idv_gpo_verification_rate_limited
     end
 
     # @param [Boolean] success
-    # @param [String] resend
-    # The Address validation letter has been requested by user
-    def idv_letter_requested(success:, resend:)
+    # @param [Hash<Key, Array<String>>] failure_reason displays GPO submission failed
+    # GPO verification submitted from Letter sent to verify address
+    def idv_gpo_verification_submitted(success:, failure_reason: nil)
       track_event(
-        :idv_letter_requested,
+        :idv_gpo_verification_submitted,
         success: success,
-        resend: resend,
+        failure_reason: failure_reason,
       )
```

Comment:
Do you mind summarizing what changed with these two methods, since GitHub decided to get, uhh, creative with the diff? Did you just reorder them, or were there any substantial changes to the structure?

Reply:
Actually, I guess it's figure-out-able enough. :shipit:

Reply:
Yeah, sorry about that - It is primarily a re-order.  I did add `gpo` to the `letter_requested` event name, too.